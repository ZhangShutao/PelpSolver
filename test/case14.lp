a1 | a2 | a3. [1]
:- K(0, 1) a1. [1]