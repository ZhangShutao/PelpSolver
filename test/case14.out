WorldView 1:
{[0.2969227424756547]:{a1},[0.2969227424756547]:{a3},[0.10923177257303593]:{},[0.2969227424756547]:{a2}}